go/kbfs/libkbfs/subscription_manager.go

```diff
@@ -286,6 +291,10 @@ func (sm *subscriptionManager) nodeChangeLocked(node Node) {
 		return
 	}
 	for _, notifier := range sm.pathSubscriptions[ref] {
+		// We are notify()-ing while holding a lock, but it's fine since the
+		// other side of the channel consumes it pretty fast, eitehr by
+		// dropping overflown ones, or by doing the actual send in a separate
```

Comment:
"overflown" isn't really the right one -- maybe just "deduplicated" or "rate-limited"?